### ISSUE TITLE
Unable to start Tika Server and get corrupt file when running tika-server.jar

### ISSUE THREAD

**Issue by devipramita** [chrismattmann/tika-python#371]
Hi, I get this error when parsing pdf using Tika
![error Tika Server](https://user-images.githubusercontent.com/23418370/167767008-3605f0f1-8b2e-498c-995f-39edf2a795de.png)

To overcome this issue, I've tried:
- setting os.environ['TIKA_SERVER_JAR'] to downloaded tika-server.jar
- setting os.environ['TIKA_PATH'] to folder of downloaded tika-server.jar
- checking on installed Java 8 (since I downloaded Tika 1.24. version) -> exist as shown below
![image](https://user-images.githubusercontent.com/23418370/167767287-c5779263-01bd-4c1f-95f4-8114c0c6da57.png)

seeing this solution https://github.com/chrismattmann/tika-python/issues/238#issuecomment-527315954
I tried to run **java -jar <<tika-server.jar>>** but it gives me another error "Invalid or corrupt jarfile tika-server.jar"

Meanwhile, the downloaded tika-server.jar contains "tika-server.jar: HTML document, ASCII text, with CRLF line terminators"

Anyone has any solution ideas to this?

Thank you

**Comment by divyaksh-shukla** [comment]
Looks like you have the wrong installation of apache-tika, rather than the jar you only have a html page. As per the current download CDN on apache tika website there are only 2 versions available: 1.28.4 & 2.4.1. You can use the below commands to download apache tika on linux
```bash
# For 1.28.4
wget https://dlcdn.apache.org/tika/1.28.4/tika-server-1.28.4.jar

# For 2.4.1
wget https://dlcdn.apache.org/tika/2.4.1/tika-server-standard-2.4.1.jar
```
You can verify if you have done the correct file download by running the following command and comparing your output to the below output:
```bash
$ file tika-server-1.28.4.jar 
tika-server-1.28.4.jar: Zip archive data, at least v2.0 to extract
```

**Comment by chrismattmann** [comment]
correct @divyaksh-shukla . Closing this one out.
